index.html

```diff
@@ -80,8 +80,8 @@ <h2 class="promo-slogan">The right ingredients<br>for the right food</h2>
       <figure class="our-mission">
         <figcaption>
           <h3 class="slogan-our-mission">Just the right food</h3>
-          <img src="gallery/decoration/decoration-2.png" class="decoration-about" alt="">
         </figcaption>
+        <img src="gallery/decoration/decoration-2.png" class="decoration-about" alt="">
```

Comment:
`alt` text is required for all images. If image not loaded user have to know what kind of image is this.

Reply:
ok